sqlmesh/core/context_diff.py

```diff
@@ -197,93 +219,93 @@ def has_snapshot_changes(self) -> bool:
 
         Returns:
-            Whether or not the node was directly modified.
+            Whether or not the snapshot was directly modified.
         """
 
-        if name not in self.modified_snapshots:
+        if s_id not in self.modified_snapshots:
             return False
 
-        current, previous = self.modified_snapshots[name]
+        current, previous = self.modified_snapshots[s_id]
         return current.fingerprint.data_hash != previous.fingerprint.data_hash
 
-    def indirectly_modified(self, name: str) -> bool:
-        """Returns whether or not a node was indirectly modified in this context.
+    def indirectly_modified(self, s_id: SnapshotId) -> bool:
+        """Returns whether or not a Snapshot ID was indirectly modified in this context.
 
         Args:
-            name: The node name to check.
+            s_id: The Snapshot ID to check.
 
         Returns:
-            Whether or not the node was indirectly modified.
+            Whether or not the Snapshot ID was indirectly modified.
         """
 
-        if name not in self.modified_snapshots:
+        if s_id not in self.modified_snapshots:
             return False
 
-        current, previous = self.modified_snapshots[name]
+        current, previous = self.modified_snapshots[s_id]
         return (
             current.fingerprint.data_hash == previous.fingerprint.data_hash
             and current.fingerprint.parent_data_hash != previous.fingerprint.parent_data_hash
         )
 
-    def metadata_updated(self, name: str) -> bool:
-        """Returns whether or not the given node's metadata has been updated.
+    def metadata_updated(self, s_id: SnapshotId) -> bool:
+        """Returns whether or not the given Snapshot ID's metadata has been updated.
 
         Args:
-            name: The node name to check.
+            s_id: The Snapshot ID to check.
 
         Returns:
-            Whether or not the node's metadata has been updated.
+            Whether or not the Snapshot ID's metadata has been updated.
         """
 
-        if name not in self.modified_snapshots:
+        if s_id not in self.modified_snapshots:
             return False
 
-        current, previous = self.modified_snapshots[name]
+        current, previous = self.modified_snapshots[s_id]
         return current.fingerprint.metadata_hash != previous.fingerprint.metadata_hash
 
-    def text_diff(self, node: str) -> str:
+    def text_diff(self, s_id: SnapshotId) -> str:
         """Finds the difference of a node between the current and remote environment.
 
         Args:
-            node: The node name.
+            s_id: The Snapshot ID.
 
         Returns:
             A unified text diff of the node.
         """
-        if node not in self.snapshots:
-            raise SQLMeshError(f"`{node}` does not exist.")
-        if node not in self.modified_snapshots:
+        if s_id not in self.snapshots:
+            raise SQLMeshError(f"`{s_id}` does not exist.")
```

Comment:
this message is now going to be more obtuse. don't you want to show the snapshot fqn?

Reply:
I tried to always use name instead  of just the id but I obviously missed this one.